testsuite/cluster-test/src/cluster_builder.rs

```diff
@@ -49,7 +49,7 @@ impl ClusterBuilderParams {
 
 pub struct ClusterBuilder {
     pub current_tag: String,
-    cluster_swarm: ClusterSwarmKube,
+    pub cluster_swarm: ClusterSwarmKube,
```

Comment:
One more interface nit - instead of making this one public, do you mind just adding `kill_job` to the ClusterSwarm interface, and then call it through context.cluster_swarm?

Reply:
How would we do `kill_job` for ClusterSwarm (not kubernetes)? To me, there's no clear mapping and *_job makes sense for only ClusterSwarmKube

Reply:
We probably will refactor it a bit, so that spawning job returns some handle, not a string. In this case job without kubernetes probably going to be a process, and handle will contain PID

Reply:
Can we push this out to later PR? I'd like to get this out soon so I can unblock client compat testing. I've created an issue for this https://github.com/libra/libra/issues/5507

Reply:
sure.
Just to make sure - we don't need to refactor string->handle right now.
I only want to move kill_job to interface